src/skip-to-main-content-link.js

```diff
@@ -0,0 +1,10 @@
+var React = require('react');
+
+var SkipToMainContentLink = React.createClass({
```

Comment:
I created this class as I was originally going to put the skip to main content link in a few different places. Ultimately I only ended up using it one place but decided to leave it here in it's own class for better re-usability.

Reply:
By adding the skip link it helps users quickly navigate over tedious navigation links so they can go straight to the main content of the page. The link is only visible when focused so only users with screen readers or tabbing through the page would reveal it:

![image](https://user-images.githubusercontent.com/2968323/71533834-04114680-28c9-11ea-9976-6126617f6095.png)